lib/hexpm_web/views/api/audit_log_view.ex

```diff
@@ -0,0 +1,7 @@
+defmodule HexpmWeb.API.AuditLogView do
+  use HexpmWeb, :view
+
+  def render("show", %{audit_log: audit_log}) do
+    Map.take(audit_log, [:action, :user_agent, :params])
```

Comment:
Do we need to render more fields for showing an `audit_log`?

Reply:
I think this is enough. Any more you had in mind?

Reply:
Nope. Let's use these two now and add more fields if needed.